src/app/screens/confirmOrdinalTransaction/index.tsx

```diff
@@ -53,12 +52,24 @@ function ConfirmOrdinalTransaction() {
     fee = BigNumber(fee);
   }
   const { id } = useParams();
-  const { data: selectedOrdinal } = useAddressInscription(id!);
-  const { setSelectedSatBundleDetails } = useSatBundleDataReducer();
+  const [txid, vout] = id!.split(':');
+  const { data: selectedOrdinal } = useAddressInscription(txid!);
```

Comment:
i think this breaks the existing identifier protocol for ordinals inscriptions.

we still want the identifier of an ordinal transaction to be the inscriptionId like 
`const inscriptionId = "c71b7653ed0be8503851f028759921223f2c2d6f52cd39aebedcd0ddbf9b8962i0";`

Reply:
https://docs.ordinals.com/inscriptions.html#inscription-ids